docs/visual-basic/language-reference/data-types/byte-data-type.md

```diff
@@ -39,6 +40,20 @@ Starting with Visual Basic 2017, you can also use the underscore character, `_`,
 
 [!code-vb[Byte](../../../../samples/snippets/visualbasic/language-reference/data-types/numeric-literals.vb#ByteS)]  
 
+Starting with Visual Basic 15.5, you can also use the underscore character (`_`) as a leading separator between the prefix and the hexadecimal, binary, or octal digits. For example:
+
+```vb
+Dim number As Byte = &H_6A
+```
+
+To use the underscore character as a leading separator, you must add the following element to your Visual Basic project (*.vbproj) file:
```

Comment:
escape the * here otherwise the style is italic for the rest of the document. Applies in multiple places in this PR. Should this text be in an include file perhaps?

Reply:
I'd thought about an include file, but since other language features require the project file element, I didn't want to create multiple ones, @mairaw. But in thinking about it again, it seems like the best thing to do.

Reply:
@rpetrusha @mairaw We have a request for a single topic describing how to manage language versions. See issue #3685  Let's move this discussion there, including the include file (see what I did there. 😆 ) and the VB language selection.